### ISSUE TITLE
Issue with DiscordSRV/Skript

### ISSUE THREAD

**Issue by StrxmyDev** [SkriptLang/Skript#6868]
### Skript/Server Version

```markdown
[11:50:09 INFO]:  [ Skript ]  Skript's aliases can be found here:  https://github.com/SkriptLang/skript-aliases 
[11:50:09 INFO]:  [ Skript ]  Skript's documentation can be found here:  https://docs.skriptlang.org/ 
[11:50:09 INFO]:  [ Skript ]  Skript's tutorials can be found here:  https://docs.skriptlang.org/tutorials 
[11:50:09 INFO]:  [ Skript ]  Server Version:  1.21-40-b45d9b6 (MC: 1.21) 
[11:50:09 INFO]:  [ Skript ]  Skript Version:  2.9.0-pre1 (skriptlang-github) 
[11:50:09 INFO]:  [ Skript ]  Installed Skript Addons:  None 
[11:50:09 INFO]:  [ Skript ]  Installed dependencies:   
[11:50:09 INFO]:  [ Skript ]   - Vault v1.7.3-b131
[11:50:09 INFO]:  [ Skript ]   - WorldGuard v7.0.11-beta1+a801a9d
[11:50:09 INFO]:  [ Skript ]   - GriefPrevention v16.18.4
```

### Bug Description

I'm unsure whether to report it to DiscordSRV or Skript, but somehow DiscordSRV is breaking Skript, despite DiscordSRV not actually support integration for Skript (not to my knowledge anyway as it was said that it would be "Impossible" in 2017 on their GIthub by a Developer)

tl;dr Skript is detecting DiscordSRV as a Skript Plugin, despite DiscordSRV (to my knowledge) not supporting integration with Skript.

### Expected Behavior

For the Code to run correctly.

### Steps to Reproduce

Install DiscordSRV & Skript on the same server and try to run literally any command via Skript.

### Errors or Screenshots

https://pastebin.com/7s4Tr3Bq

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by EquipableMC** [comment]
`java.lang.IllegalStateException: zip file closed` Have you reloaded the server with /reload?

**Comment by StrxmyDev** [comment]
> `java.lang.IllegalStateException: zip file closed` Have you reloaded the server with /reload?

Yes I have prior, is this what's causing the issue 🤣

**Comment by StrxmyDev** [comment]
Yes I have as I had to remove some Skript Addons that were also causing this error, is this what's causing the issue this whole time and i've uninstalled the addons for no reason🤣

**Comment by EquipableMC** [comment]
Yeah don't use /reload, it is buggy and breaks skript. Just restart your server

**Comment by APickledWalrus** [comment]
As other's have stated, this is caused by using /reload which is not supported by Skript.
